src/minimega/qos_cli.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+	"minicli"
+	"strconv"
+)
+
+var qosCLIHandlers = []minicli.Handler{
+	{
+		HelpShort: "add qos constraints to an interface",
+		HelpLong:  "Fritz Fritz Fritz",
+		Patterns: []string{
```

Comment:
I'd update the patterns to support finding interfaces better and change some semantics:

    qos show [vm or interface]
    qos <add,> <vm> <interface index> <loss,> <percent>
    qos <add,> <vm> <interface index> <delay,> <ms>
    qos <add,> <interface> <loss,> <percent>
    qos <add,> <interface> <delay,> <ms>
    clear qos
    clear qos <vm> <interface index>
    clear qos <interface>

@jcrussell should comment